GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -960,101 +949,13 @@ void ShowRevisions()
 
         private void OnActivate()
         {
-            CheckForMergeConflicts();
-            return;
+            ThreadHelper.ThrowIfNotOnUIThread();
```

Comment:
Everything on this point onwards to `ThreadHelper.JoinableTaskFactory.RunAsync(...)` had to run on UI thread. Now all of that functionality is gone from here, so this check is now redundant.